PointOfInteraction/SmartPointOfInteraction/doc/spec.md

```diff
@@ -0,0 +1,73 @@
+# SmartPointOfInteraction
+
+## Description
+
+A Smart Point of Interaction defines a place with technology to interact with citizens, for example, through Beacon technology from Apple, Eddystone/Physical-Web from Google or other proximity-based interfaces. Since the interactive area could be composed by more than one device providing the technology, this model represents a group of Smart Spot devices.
+
+The data model includes information regarding the area/surface covered by the technology (i.e., the area covered by Bluetooth Low Energy-based Beacon or Eddystone), the schedule of the broadcasted information, links to the multimedia resources / Web Apps etc. In addition, the data model may have a reference to another OMA NGSI entity such as a Parking, a Point of Interest (POI), ... which will be improved with this technology to interact.
```

Comment:
comma missed between Web Apps and etc. 
etc. missed between (POI), removing the "..." 
with enriched interaction provided by this technology.